client/reader/following-edit/subscribe-form.jsx

```diff
@@ -129,11 +160,23 @@ var FollowingEditSubscribeForm = React.createClass( {
 		}
 
 		if ( showSearchResult ) {
-			searchResult = ( <FollowingEditSubscribeFormResult
-				isValid={ isWellFormedFeedUrl }
-				url={ searchString }
-				onFollowToggle={ handleFollowToggle } />
-			);
+			if ( ! config.isEnabled( 'reader/follow-feed-search' ) ) {
+				searchResult = ( <FollowingEditSubscribeFormResult
+					isValid={ isWellFormedFeedUrl }
+					url={ searchString }
+					onFollowToggle={ handleFollowToggle } />
+				);
+			} else if ( this.state.newFeeds ) {
+				console.log( this.state.newFeeds );
+				searchResult = this.state.newFeeds.map( function( item ) {
+				 		return ( <div className="subscription-list-item__header-content"
```

Comment:
I suspect the issue for showing multiple results is here combined with the absolute positioning at line 145 of style.scss in the same dir. By removing this extra div and disabling the position:absolute in my browser I go multiple results to show up (ugly but maybe workable and gets you on the right path?).